src/index.html

```diff
@@ -11,6 +11,57 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      class="form"
+      data-qa="big"
+
+    >
+
+      <label class="box">
+
+        <input
+        type="search"
+        name="search"
+        placeholder="Try &quot;Los Angeles&quot;"
+        class="form__input"
+        data-qa="keypress"
+
+      >
+
+        <img
+        class="form__img"
+        src="./images/Search.svg"
+        alt="search"
+
+       >
+      </label>
+
+    </form>
+
+    <form
```

Comment:
Try to format it

Reply:
Check this checklist
https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md

Reply:
I agree, do it